scenarios.md

```diff
@@ -0,0 +1,225 @@
+
+1. The Wabbit Networks company builds their netmonitor software. As a result of the build, they produce an [OCI Image][oci-image], a Software Bill of Materials (`SBoM`) and to comply with gpl licensing, produce another artifact which contains the source (`src`) to all the gpl licensed projects.  
+In addition to the `image`, `SBoM` and `src` artifacts, the build system produces an [OCI Index][oci-index] that encompassed the three artifacts.  
+Each of the artifacts, and the `index` are signed with Notary v2.  
+1. The index and its signed contents are pushed to a public OCI compliant registry.
+1. ACME Rockets consumes the netmonitor software, importing the index and its referenced artifacts into their private registry.
+1. The ACME Rockets environment enforces various company policies prior to any deployment, evaluating the content in the `SBoM`. The policy manager trusts the content within the SBoM is accurate, because they trust artifacts signed with the wabbit-networks key. The `src` content isn't evaluated at deployment time and can be left within the registry.
+1. Once the policy manager completes its validation, the deployment to the hosting environment is initiated. The `SBoM` is no longer needed, allowing the `image` to be deployed separately. A `deploy` artifact, referencing a specific configuration definition, may also be signed and saved, providing a historical record of what was deployed. The hosting environment also validates content is signed by trusted entities.
+
+**Implications of this requirement:**
+
+- Signatures can be placed on any type of artifact stored in an OCI compliant registry using an [OCI Manifest][oci-manifest]
+- Signatures can be placed on an [OCI Index][oci-index], allowing a entity to define a collection of artifacts.
+- Signatures and their public keys can be moved within, and across OCI compliant registries which support Notary v2.
+- Because content is trusted, an ecosystem of other projects and products can leverage information in various formats.
+
+### Scenario #1: Local Build, Sign, Validate
+
+Prior to committing any code, a developer can test the: "build, sign, validate scenario"
+
+1. Locally build a container image using a non-registry specific `name:tag`, such as:  
+  `$ docker build net-monitor:dev`
+1. Locally sign `net-monitor:dev` 
+1. Run the image on the developers local machine which is configured to only accept signed images. 
+  `$ docker run net-monitor:dev`
+
+**Implications of this requirement:**
+
+- The developer has access to signing keys. How they get the keys is part of the usability spec.
+- The local environment has a policy by which it states the set of keys it accepts.
+- The signing and validation of artifacts does not require a registry. The local host can validate the signature using the public keys it accepts.
+- The key used for validation may be hosted in a registry, or other accessible location.
```

Comment:
This feels more like you're proposing a solution than talking about scenarios / requirements.  Is this appropriate for the document?  

Implications in general seems to be a mix of actual implications / requirements and details from a proposed implementation intertwined.   Can we separate this out so that it is just requirements?  It's weird to read about details of a proposed implementation with no description.